addons/website_slides_survey/views/survey_survey_views.xml

```diff
@@ -25,4 +25,21 @@
             </p>
         </field>
     </record>
+
+    <record id='survey_survey_view_form' model="ir.ui.view">
+        <field name="name">survey.survey.view.form</field>
+        <field name="model">survey.survey</field>
+        <field name="inherit_id" ref="survey.survey_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//button[@name='action_survey_user_input_certified']" position="before">
+                <button name="action_survey_view_courses"
+                    type="object"
+                    class="oe_stat_button"
+                    attrs="{'invisible': [('courses_count', '=', 0)]}"
+                    icon="fa-graduation-cap">
+                    <field string="Courses" name="courses_count" widget="statinfo"/>
+                </button>
+            </xpath>
+        </field>
+    </record>
 </odoo>
```

Comment:
Not you fault, but can you add an extra line here while you're at it?

Reply:
Done